dashboard/app/helpers/levels_helper.rb

```diff
@@ -91,7 +91,7 @@ def select_and_remember_callouts(always_show = false)
   # Options hash for all level types
   def app_options
     # Provide the channel for templated and applab levels.
-    set_channel if @level.try(:project_template_level) || @level.game == Game::APPLAB
+    set_channel if @level.try(:project_template_level) || @level.game == Game.applab
```

Comment:
this is fixing an existing bug? maybe add a test to avoid regressions?
also, why is try necessary for project_template_level but not game (at first glance it seems to me that it's not necessary for either)?